products/views.py

```diff
@@ -1,3 +1,23 @@
 from django.shortcuts import render
 
 # Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from products.models import *
+
+class RecommendedView(View):
+    def get(self, request, product_id):
+        try:
+            category_id         = Product.objects.get(id = product_id).category
+            recommend_list = [{
+                'name'      : value.name,
+                'image'     : [image.url for image in value.productimage_set.all()],
+                'skintype'  : [types.skin_type.name for types in value.productskintype_set.all()]
+                } for value in Product.objects.filter(category = category_id).exclude(id=product_id)]
```

Comment:
해당 값은 단순 `value`가 아니라 `product`입니다. 변수명을 정확히 작성해주세요.
```suggestion
            recommend_list = [{
                'name'      : product.name,
                'image'     : [image.url for image in product.productimage_set.all()],
                'skintype'  : [types.skin_type.name for types in product.productskintype_set.all()]
                } for product in Product.objects.filter(category = category_id).exclude(id=product_id)]
```